techniques/client-side-script/SCR30.html

```diff
@@ -22,10 +22,10 @@
 function doExpand() {
 	var links = document.links;
 	
-	for (link of links) {
```

Comment:
could probably just change this to a `links.forEach(link) {` without touching the rest of the code

Reply:
actually, just tried, and seems `links` isn't a proper array but a collection...so yeah, just ignore. better to just be explicit with a for loop...

Reply:
Yeah, I had no luck w/ `forEach` either, but I did just push another commit that hopefully makes the overall change a bit more terse.